roles/telemetry_verify_metrics/README.md

```diff
@@ -0,0 +1,50 @@
+telemetry_verify_metrics
+=========
+
+Test that expected metrics appear in Prometheus
+
+Requirements
+------------
+OpenStack deployed with the following enabled:
+- telemetry
+- metricstorage
+- ceilometer
+- rabbitmq
+
+Tests:
+------
+- Verify OpenStack is deployed correctly
+    - Verify telemetry is ready
+    - Verify metricstorage is ready
+    - Verify ceilometer is ready
+    - Verify rabbitmq is ready
+- Verify RabbitMQ metrics are being exposed and stored
+    - Check the rabbitmq metrics endpoint
+    - Use openstack observabilityclient to verify RabbitMQ metrics are stored in Prometheus
+- Verify Ceilometer metrics are being exposed and stored
+    - Use openstack observabilityclient to verify Ceilometer central metrics are stored in Prometheus
+    - Use openstack observabilityclient to verify Ceilometer compute metrics are stored in Prometheus
+- Verify NodeExporter metrics are being exposed and stored
+    - Use openstack observabilityclient to verify NodeExporter metrics are stored in Prometheus
+
+Role Variables
+--------------
+openstack\_cmd - command to access openstack cli. For example: "oc rsh openstackclient openstack"
```

Comment:
metric_sources_to_test should be included here too, with the rename of the variable